presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1172,6 +1173,12 @@ public SystemSessionProperties(
                         "Enable streaming for partial aggregation",
                         featuresConfig.isStreamingForPartialAggregationEnabled(),
                         false),
+                booleanProperty(
+                        PREFER_MERGE_JOIN,
+                        "Prefer merge join for sorted join inputs, e.g., tables pre-sorted, pre-partitioned by join columns." +
+                                "Specifically for hive connectors, it also requires order_based_execution_enabled to make it work.",
```

Comment:
Originally it's written in that way and I mentioned the same logic to @yuanzhanhku, but he had strong opinion on adding the hive connector note, see the discussion above (I just un-resolve the comment above)

@yuanzhanhku let's make a final decision here on how the note should be?

Reply:
Since it requires two session properties to make it work for hive connector. I think it is better to document this somewhere. Otherwise, people will not be able to figure it out. Maybe have a post when this feature is ready or document it in some code comment is also fine.  If we can find a better place to document this, we can remove it here.

Reply:
Maybe a good compromise would be a generic note about connectors exposing sortedness here, and when the feature is implemented we can add formal documentation for how to use it.

Reply:
Sounds good to me.